core/in-app-update/src/main/java/com/tgyuu/inappupdate/InAppUpdateManager.kt

```
@@ -0,0 +1,64 @@
+                    appUpdateManager.startUpdateFlowForResult(
+                        appUpdateInfo,
+                        activity,
+                        AppUpdateOptions.newBuilder(updateType).build(),
+                        REQUEST_CODE_UPDATE
+                    )
+                }
+                else -> {
+                    context.openPlayStore()
+                }
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)